TRACONBoundaries.geojson

```diff
@@ -502,12 +502,13 @@
     {"type":"Feature","properties":{"id":"UNNT","prefix":["UNNT_R"],"name":"Novosibirsk Radar"},"geometry":{"type":"MultiPolygon","coordinates":[[[[82.96000,55.43083],[82.90972,55.44778],[82.83167,55.46417],[82.75083,55.47556],[82.66833,55.48222],[82.58528,55.48389],[82.50222,55.48028],[82.42028,55.47222],[82.34028,55.45889],[82.26333,55.44111],[82.19028,55.41861],[82.12167,55.39194],[82.05861,55.36139],[82.00139,55.32722],[81.95111,55.28972],[81.90806,55.24944],[81.87250,55.20667],[81.84528,55.16222],[81.82611,55.11639],[81.81556,55.06944],[81.81389,55.02250],[81.82056,54.97528],[81.83583,54.92889],[81.85944,54.88389],[81.89083,54.84028],[81.93000,54.79889],[81.97639,54.76000],[82.02972,54.72417],[82.08889,54.69167],[82.15361,54.66306],[82.22306,54.63833],[82.29639,54.61778],[82.37306,54.60194],[82.45222,54.59056],[82.53278,54.58417],[82.61417,54.58250],[82.69528,54.58583],[82.69500,54.58806],[82.96667,54.68306],[83.00000,54.75000],[83.19167,54.80167],[83.30306,55.11806],[83.28306,55.11667],[82.98306,55.06667],[83.00000,55.10000],[83.01667,55.15000],[82.95000,55.23306],[82.91667,55.25000],[82.86667,55.31667],[82.96000,55.43083]]]]}},
     {"type":"Feature","properties":{"id":"UNKL","prefix":["UNKL"],"name":"Krasnoyarsk Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[92.70000,57.04500],[92.28306,57.03806],[91.71639,56.75056],[91.13306,56.28389],[91.23306,55.98389],[91.11639,55.51694],[91.54944,55.41694],[92.46639,55.38389],[92.99139,55.34889],[93.48306,55.82500],[93.75000,55.86667],[94.00000,56.00000],[93.30000,56.12000],[93.11667,56.15000],[93.18306,56.41306],[93.58000,56.61667],[93.55194,56.73556],[93.16639,56.90056],[92.70000,57.04500]]]]}},
+    {"type":"Feature","properties":{"id":"UNKL_R","prefix":["UNKL"],"name":"Krasnoyarsk Radar"},"geometry":{"type":"MultiPolygon","coordinates":[[[[93.21639,55.97111],[93.30000,56.12000],[93.11667,56.15000],[93.18306,56.41306],[93.12000,56.45833],[93.06250,56.49333],[92.99861,56.52444],[92.92917,56.55194],[92.85472,56.57528],[92.77611,56.59417],[92.69444,56.60833],[92.61028,56.61750],[92.52500,56.62194],[92.43917,56.62111],[92.35417,56.61556],[92.27056,56.60500],[92.18944,56.58972],[92.11194,56.56972],[92.03861,56.54528],[91.97056,56.51667],[91.90833,56.48444],[91.85250,56.44861],[91.80389,56.41000],[91.76306,56.36861],[91.73056,56.32500],[91.70611,56.27972],[91.69056,56.23333],[91.68389,56.18639],[91.68611,56.13917],[91.69694,56.09250],[91.71667,56.04639],[91.74472,56.00194],[91.78083,55.95944],[91.82472,55.91917],[91.87583,55.88167],[91.87278,55.88028],[91.87278,55.88028],[91.93444,55.84667],[91.99833,55.81583],[92.06722,55.78917],[92.14083,55.76639],[92.21806,55.74806],[92.29806,55.73417],[92.38028,55.72528],[92.46389,55.72111],[92.54750,55.72194],[92.63083,55.72750],[92.71250,55.73778],[92.79194,55.75306],[92.86806,55.77278],[92.94028,55.79667],[93.00778,55.82472],[93.06972,55.85667],[93.12556,55.89167],[93.17472,55.93000],[93.21639,55.97111]]]]}},
```

Comment:
This is malformed. The ID should not contain an underscore. 

I'm also very curious to why there are added in the first place. What is the difference between UNKL_APP and UNKL_R_APP?

Reply:
UNKL_R_APP and UNKL_APP are different dispatch positions. UNKL_R_APP radar, UNKL_APP approach

Reply:
> UNKL_R_APP and UNKL_APP are different dispatch positions. UNKL_R_APP radar, UNKL_APP approach

Right, but would UNKL_R_APP ever be online without UNKL_APP being online?

Reply:
> This is malformed. The ID should not contain an underscore.
> 
> I'm also very curious to why there are added in the first place. What is the difference between UNKL_APP and UNKL_R_APP?

I wrote about it. In Russia there are special ATC position Radar. It can be online without approach.

Reply:
> UNKL_R_APP and UNKL_APP are different dispatch positions. UNKL_R_APP radar, UNKL_APP approach

Use {"id":"UNKL","prefix":["UNKL_R"],"name":"Krasnoyarsk Radar"}
Then it will work correctly

Reply:
ID this is what will be written above boundary on map. Prefix is about ATC callsign.